server/handlers.go

```diff
@@ -40,14 +42,19 @@ func fileHandler(w http.ResponseWriter, r *http.Request) {
 	pathParser(w, path)
 }
 
+func triggerClone() {
+
+}
```

Comment:
empty function

Reply:
changed the `main` structure on the whole